minemeld/ft/json.py

```diff
@@ -73,6 +79,9 @@ def configure(self):
         self.username = self.config.get('username', None)
         self.password = self.config.get('password', None)
 
+        self.header_key = self.config.get('header_key', None)
```

Comment:
it would be better to load a full dictionary of header/header value from the config to let users specify multiple headers. Something like:
`self.headers = self.config.get('headers', None)`